specification/quota/resource-manager/Microsoft.Quota/stable/2021-03-15/examples/getComputeUsages.json

```diff
@@ -0,0 +1,159 @@
+{
+  "parameters": {
+    "scope": "subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.Compute/locations/eastus",
+    "api-version": "2021-03-15"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
```

Comment:
Does this api return quotaLimits resources created by the user only or it returns all by default representing default quotas?

Reply:
Current quota Limits for the resource.
The user checks at first -
1. What's my quota limit.
2. Based on the user's requirements, if new quota is required - Current limit is 15, currentUsages = 10, user's deployment needs 4 more VCPU, =14, No action needed. 
3. If the user deployment needs 10 more VCPUs, so newLimit = currentUsages+10 = 10+10 =20
4. We have documented it, how to use the Quota APIs. Users can increase quota limit in advance, if 80% of quota is used. Different Orgs can program based on their rules.